src/components/AudioCell.vue

```diff
@@ -151,6 +160,31 @@ export default defineComponent({
       store.dispatch(STOP_AUDIO, { audioKey: props.audioKey });
     };
 
+    //長い文章をコピペしたときに句点（。）で自動区切りする
+    //https://github.com/Hiroshiba/voicevox/issues/25
+    const pasteOnAudioCell = async (evt: ClipboardEvent) => {
+      if (evt.clipboardData) {
+        //"。"で区切り
```

Comment:
コメントと実装が少しだけ乖離していそうです。
```suggestion
        //"。"と改行で区切り
```